This is test cases for 3-say_my_name.py func
=============================================

The module has 1 function called: say_my_name and it takes 2 args
namely: first name and last name. Both of the args must be a string
or it will raise TypeError

Doc
===
The documentation of both module and function must be verified for ALX req

	>>> mod_doc = __import__('3-say_my_name').__doc__
	>>> print(len(mod_doc) > 0)
	True

	>>> fun_doc = __import__('3-say_my_name').say_my_name.__doc__
	>>> print(len(fun_doc) > 0)
	True

For some general cases
======================

	>>> say_my_name = __import__('3-say_my_name').say_my_name

	>>> say_my_name("Kimba", "ALX")
	My name is Kimba ALX

	>>> say_my_name("BettyChecker")
	My name is BettyChecker 

	>>> say_my_name("Agent", "47")
	My name is Agent 47

Passing None:

	>>> say_my_name("School", None)
	Traceback (most recent call last):
	TypeError: last_name must be a string

	>>> say_my_name(None, "Program")
	Traceback (most recent call last):
	TypeError: first_name must be a string

Passing in data types other than string:

	>>> say_my_name(43, "344")
	Traceback (most recent call last):
	TypeError: first_name must be a string

	>>> say_my_name("0933", 322)
	Traceback (most recent call last):
	TypeError: last_name must be a string

	>>> say_my_name(["John"], "Smith")
	Traceback (most recent call last):
	TypeError: first_name must be a string

Missing args or more than 2 args passed:

	>>> say_my_name()
	Traceback (most recent call last):
	TypeError: say_my_name() missing 1 required positional argument: 'first_name'

	>>> say_my_name("John", "ALX", "Program")
	Traceback (most recent call last):
	TypeError: say_my_name() takes from 1 to 2 positional arguments but 3 were given
